assets/src/application/services/apollo/mutations/useEntityMutation.ts

```diff
@@ -81,8 +81,8 @@ const useEntityMutation = (type: EntityType, id?: string): EntityMutation => {
 			// e.g. "createDatetime", "updateTicket"
 			const mutationName = `${mutationType.toLowerCase()}Espresso${type}`;
 			// Example result: { data: { deletePrice: { price : {...} } } }
-			const path: string[] = ['data', mutationName, `espresso${type}`];
-			const entity: any = pathOr({}, path, result);
+			const path = ['data', mutationName, `espresso${type}`];
+			const entity = pathOr<any>({}, path, result);
```

Comment:
can also do something like 

```ts
pathOr<T extends Entity>({}, path, result);
```

if there is a common sub-type expected

Reply:
Better